src/ast/code_generator.ml

```diff
@@ -745,8 +678,19 @@ let add_owi_funcs (m : modul) : modul * (string * int) array =
   in
   update_func ()
 
-let generate (m : modul) : modul Result.t =
-  let m, owi_funcs = add_owi_funcs m in
+let generate (symbolic : bool) (m : modul) : modul Result.t =
+  let owi_funcs =
+    if symbolic then
+      [| ("i32_symbol", ([], [ Num_type I32 ]))
+       ; ("i64_symbol", ([], [ Num_type I64 ]))
+       ; ("f32_symbol", ([], [ Num_type F32 ]))
+       ; ("f64_symbol", ([], [ Num_type F64 ]))
+       ; ("assume", ([ (None, Num_type I32) ], []))
```

Comment:
Are all these functions actually required by the generated code?

Reply:
OK so, they can probably removed. We'll re-introduce them if we start generating more complicated RAC stuff.

Reply:
ok